content/en/security/application_security/software_composition_analysis/_index.md

```diff
@@ -109,3 +126,4 @@ Software Composition Analysis enriches the information APM is already collecting
 [10]: /account_management/rbac/permissions/#integrations
 [11]: /integrations/jira/
 [12]: https://app.datadoghq.com/security/configuration/asm/setup
+[13]:https://github.com/DataDog/guarddog
```

Comment:
```suggestion
[13]: https://github.com/DataDog/guarddog
```
^ This will still work fine, but we should add the space for consistency's sake.